test/Base_Tests/src/Data/Numbers_Spec.enso

```diff
@@ -565,6 +571,16 @@ add_specs suite_builder =
             3.truncate . should_equal 3
             -3.truncate . should_equal -3
 
+    suite_builder.group "Hashing" group_builder->
+        # TODO: Pending https://github.com/enso-org/enso/issues/9296.
+        group_builder.specify "Hash consistency for values near Double.MIN/MAX_VALUE" pending="hash consistency" <|
+            ok_values = [9223372036854775000, -9223372036854775000]
+            bad_values = [9223372036854776000, -9223372036854776000]
```

Comment:
I believe these are the only two values that have this problem. This is a separate task because I want to make sure that fixing it doesn't introduce inconsistencies with other code that might have this problem.

Reply:
What is the difference between `ok_values` and `bad_values`? I'm not sure if I understand the test. But it's a separate task so probably OK for now.